### PR TITLE
Add New Port OverGun

### PR DESCRIPTION
**Pull request by MadShmupper**
# New Port for { OverGun }

## Game Information
- **Title**: { OverGun }
- **URL**: { https://yousfiwalid.itch.io/over-gun }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — DISMISSED** [review]

File: ports/overgun/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<gameList>
+  <game>
+    <path>./OverGun.sh</path>
+    <name>OverGun</name>
+    <desc>OverGun is a shoot'em up 2D platform where your character’s weapon is bigger than he can handle! Take control of the crazy pixeled shooter to help him keep control of his gun and bullets through the dungeon.</desc>
+    <releasedate>20200712T000000</releasedate>
+    <developer>YOUSFI M WALDI</developer>
+    <publisher>YOUSFI M WALDI</publisher>
+    <genre>platformer/strategy</genre>
+    <image>./overGun/cover.png</image>
```
Comment (Cebion) [inline ports/overgun/gameinfo.xml:11]: <image> tag folderpath is wrong. Needs to match the folder (linux is case sensitive)

File: ports/overgun/overgun/overgun.gptk [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+down = s
+left = a
+right = d
+
+left_analog_up = \"
+left_analog_down = s
+left_analog_left = a
+left_analog_right = d
+
+a = cntrl
```
Comment (Cebion) [inline ports/overgun/overgun/overgun.gptk:14]: There is no keyboard key called "cntrl" only "ctrl"

**Review by Cebion — CHANGES_REQUESTED** [review]
See comments

**Review by kloptops — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Comment by MadShmupper** [comment]
👾👾👾
       : 
   🚀

**Pull request merged**